fastai/metrics.py

```diff
@@ -238,17 +238,34 @@ class KappaScore(ConfusionMatrix):
     """
     Compute the rate of agreement (Cohens Kappa).
     Ref.: https://github.com/scikit-learn/scikit-learn/blob/bac89c2/sklearn/metrics/classification.py
+    
```

Comment:
Please put this in the documentation notebook `metrics.ipynb` instead, we like the doc string to be as short as possible (fastai style guide).

Reply:
I left the first line and moved the sklearn reference to `metrics.ipynb`. For consistency, I did the same with the sklearn reference in `MatthewsCorreff` in `metrics.py`.

I added discussion of parameters and deployment to `metrics.ipynb`.